2024-03-05 06:21:30,757 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-05 06:21:30,760 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Configure stats pid to 25648
2024-03-05 06:21:30,761 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-05 06:21:30,761 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-05 06:21:30,761 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-05 06:21:30,761 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-05 06:21:30,762 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-05 06:21:30,765 INFO    MainThread:25648 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_062130-ojm03uxl\logs\debug.log
2024-03-05 06:21:30,767 INFO    MainThread:25648 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_062130-ojm03uxl\logs\debug-internal.log
2024-03-05 06:21:30,768 INFO    MainThread:25648 [wandb_init.py:init():566] calling init triggers
2024-03-05 06:21:30,768 INFO    MainThread:25648 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-05 06:21:30,769 INFO    MainThread:25648 [wandb_init.py:init():591] re-initializing run, found existing run on stack: hgxqhtze
2024-03-05 06:21:30,771 INFO    MainThread:25648 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier2/hgxqhtze
2024-03-05 06:21:30,771 INFO    MainThread:25648 [jupyter.py:save_history():445] not saving jupyter history
2024-03-05 06:21:30,772 INFO    MainThread:25648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-05 06:21:30,772 INFO    MainThread:25648 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-05 06:21:30,773 INFO    MainThread:25648 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-05 06:21:30,777 INFO    MainThread:25648 [wandb_run.py:_restore():2193] restore
2024-03-05 06:21:30,777 INFO    MainThread:25648 [wandb_run.py:_restore():2199] restore done
2024-03-05 06:21:36,885 INFO    MainThread:25648 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-05 06:21:36,886 INFO    MainThread:25648 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-05 06:21:36,915 INFO    MainThread:25648 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-05 06:21:36,961 INFO    MainThread:25648 [wandb_init.py:init():616] starting backend
2024-03-05 06:21:36,962 INFO    MainThread:25648 [wandb_init.py:init():620] setting up manager
2024-03-05 06:21:37,002 INFO    MainThread:25648 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-05 06:21:37,003 INFO    MainThread:25648 [wandb_init.py:init():628] backend started and connected
2024-03-05 06:21:37,083 INFO    MainThread:25648 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-05 06:21:37,089 INFO    MainThread:25648 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 3806: character maps to <undefined>
2024-03-05 06:21:37,090 INFO    MainThread:25648 [wandb_init.py:init():720] updated telemetry
2024-03-05 06:21:37,201 INFO    MainThread:25648 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-05 06:21:38,587 INFO    MainThread:25648 [wandb_run.py:_on_init():2262] communicating current version
2024-03-05 06:21:38,851 INFO    MainThread:25648 [wandb_run.py:_on_init():2271] got version response 
2024-03-05 06:21:38,851 INFO    MainThread:25648 [wandb_init.py:init():804] starting run threads in backend
2024-03-05 06:21:39,173 INFO    MainThread:25648 [wandb_run.py:_console_start():2241] atexit reg
2024-03-05 06:21:39,174 INFO    MainThread:25648 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-05 06:21:39,174 INFO    MainThread:25648 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-05 06:21:39,178 INFO    MainThread:25648 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-05 06:21:39,183 INFO    MainThread:25648 [wandb_init.py:init():847] run started, returning control to user process
2024-03-05 06:21:59,931 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-05 06:21:59,934 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Configure stats pid to 25648
2024-03-05 06:21:59,934 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-05 06:21:59,934 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-05 06:21:59,935 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-05 06:21:59,935 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-05 06:21:59,935 INFO    MainThread:25648 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-05 06:21:59,938 INFO    MainThread:25648 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_062159-bau4yut5\logs\debug.log
2024-03-05 06:21:59,939 INFO    MainThread:25648 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_062159-bau4yut5\logs\debug-internal.log
2024-03-05 06:21:59,940 INFO    MainThread:25648 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000202276C1100>
2024-03-05 06:21:59,940 INFO    MainThread:25648 [wandb_init.py:init():566] calling init triggers
2024-03-05 06:21:59,940 INFO    MainThread:25648 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-05 06:21:59,941 INFO    MainThread:25648 [wandb_init.py:init():591] re-initializing run, found existing run on stack: ojm03uxl
2024-03-05 06:21:59,943 INFO    MainThread:25648 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier2/ojm03uxl
2024-03-05 06:21:59,944 INFO    MainThread:25648 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-05 06:21:59,947 INFO    MainThread:25648 [wandb_run.py:_restore():2193] restore
2024-03-05 06:21:59,947 INFO    MainThread:25648 [wandb_run.py:_restore():2199] restore done
2024-03-05 06:22:06,362 INFO    MainThread:25648 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-05 06:22:06,363 INFO    MainThread:25648 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-05 06:22:06,387 INFO    MainThread:25648 [wandb_run.py:_footer_sync_info():3825] logging synced files
